### ISSUE TITLE
DefaultOption object is ignored when used inside a @Value enum decorator

### ISSUE THREAD

**Issue by thiago-soliveira** [indigotech/graphql-schema-decorator#36]
The `description` field is ignored when used inside `@Value` decorators like below.

```typescript
import { Description, EnumType, Value } from 'graphql-schema-decorator';

@EnumType({ description: 'Administration place kind enum' })
export class AdministrationPlaceKind {

    @Value('PUBLIC', { description: 'Public administration places' })
    PUBLIC: string;

    @Value('PRIVATE', { description: 'Private administration places' })
    PRIVATE: string;

}
```

**Comment by felipesabino** [comment]
@thiago-soliveira is this for `0.8.0`?

**Comment by thiago-soliveira** [comment]
@felipesabino Yes.

**Comment by felipesabino** [comment]
@thiago-soliveira I could not reproduce after my refactor on `@Field` decorator so I will mark it as solved when #34 is merged

My test schema was declared as

```

@EnumType({ description: 'Administration place kind enum' })
class AdministrationPlaceKind {

    @Value('PUBLIC', { description: 'Public administration places' })
    PUBLIC: string;

    @Value('PRIVATE', { description: 'Private administration places' })
    PRIVATE: string;

}
@ObjectType()
class QueryType {
    @Field()
    greeting(
      @Arg({name: 'arg', type: AdministrationPlaceKind}) arg: any,
    ): string {
        return `Hello, ${arg}!`;
    }
}

@Schema()
class SchemaType {
    @Query() query: QueryType;
}
```

And the resulting schema has the proper descriptions for enums:

```
schema {
  query: Queries
}

# Administration place kind enum
enum AdministrationPlaceKind {
  # Public administration places
  PUBLIC

  # Private administration places
  PRIVATE
}

# Reads from the backend
type Queries {
  greeting(arg: AdministrationPlaceKind): String
}
```
